package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "webpack --node-env=production --progress",
     "build:compatibility": "webpack --env mode=browser --node-env=production --progress",
+    "build:install": "npm i --lockfile-version 1 --package-lock-only && npm run build",
```

Comment:
I believe we want to be on lockfile 2, should be fixed in Dash and also in my PR

Reply:
ok updated!